src/Nix/Builtins.hs

```diff
@@ -41,6 +41,7 @@ import           Data.Coerce
 import           Data.Fix
 import           Data.Foldable (foldrM)
 import qualified Data.HashMap.Lazy as M
+import qualified Data.HashSet as HS
```

Comment:
I'd prefer if the qualified name was just `S`, and so didn't encode anything about which *kind* of Set is being used, since we might change that in future.